pkg/pods/pod.go

```diff
@@ -306,12 +318,33 @@ func (pod *Pod) currentPodManifestPath() string {
 	return filepath.Join(pod.path, "current_manifest.yaml")
 }
 
-func (pod *Pod) ConfigDir() string {
-	return filepath.Join(pod.path, "config")
+func (pod *Pod) EnvDir(manifest Manifest) (string, error) {
+	return pod.PathFor("env", manifest)
+}
+
+func (pod *Pod) ConfigPath(manifest Manifest) (string, error) {
+	return pod.PathFor(CONFIG_FILE_NAME, manifest)
+}
+
+func (pod *Pod) PlatformConfigPath(manifest Manifest) (string, error) {
+	return pod.PathFor(PLATFORM_CONFIG_FILE_NAME, manifest)
+}
+
+func (pod *Pod) PathFor(name string, manifest Manifest) (string, error) {
+	namespacedDir, err := pod.NamespacedPath(manifest)
+	if err != nil {
+		return "", util.Errorf("Unable to find path for %s: %s", name, err)
+	}
+
+	return filepath.Join(namespacedDir, name), nil
 }
 
-func (pod *Pod) EnvDir() string {
-	return filepath.Join(pod.path, "env")
+func (pod *Pod) NamespacedPath(manifest Manifest) (string, error) {
```

Comment:
open to suggestions for this function name and how to refer to this directory in general